packages/synthetix-main/contracts/storage/Market.sol

```diff
@@ -324,6 +325,19 @@ library Market {
         return self.capacity < getLockedLiquidity(self);
     }
 
+    /**
+     * @dev Gets any outstanding debt. Do not call this method except in tests
+     *
+     * TODO: Understand distributeDebt() first.
+     */
+    function getOutstandingDebt(Data storage self, uint128 poolId) internal returns (int debtChange) {
+        return int128(self.pools[poolId].pendingDebt) + self.debtDist.updateActorShares(bytes32(uint(poolId)), 0);
```

Comment:
Please use SafeCast here 🙏 
Even in the second case, in poolId, it might be annoying to use SafeCast, but I want to promote the rule of never doing low level casting in the code, except in SafeCast of course.